src/components/menu/__tests__/menu-popover.test.tsx

```diff
@@ -1,5 +1,6 @@
 import { fireEvent, render } from '@testing-library/react'
 import { Menu } from '../menu'
+import { menuButtonKeyDownHandler, menuItemKeyDownHandler, menuListFocusOutHandler } from '../menu-popover'
```

Comment:
**suggestion:** Personally, I think the desire to test these handlers independently from the menu popper component suggests they should be in their own module.

I also wonder how much value there is in testing these handlers in a solitary (non-integrated) manner 🤔

Reply:
actually that pattern is to make the test easier because currently we don't install `@testing-library/jest-dom` (which I think we need it e.g: to test focus called), and also it avoiding snapshot test (eg: testing visiblity of menu). by using handler like that I can mock the handler and test it easily.

Reply:
Ahh, yeah okay. I keep forgetting that restriction still exists 🤦‍♂️ I'll see if I can get buy-in for #158 and #159 from Will and Adrian so that we can start leveraging React Testing Library properly.